settings.gradle

```diff
@@ -16,12 +17,5 @@ pluginManagement {
 }
 
 plugins {
-    id 'com.diffplug.blowdryerSetup' version '1.7.1'
-    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.7.0' // Provides java toolchains
-}
-
-blowdryerSetup {
-    repoSubfolder 'gtnhShared'
-    github('GTNewHorizons/ExampleMod1.7.10', 'tag', '0.2.2')
-    //devLocal '.' // Use this when testing config updates locally
```

Comment:
Looking at the gradle plugin - it looks like everything is coming from the local repo for spotless settings, instead of the central repo now?

Reply:
(Which is probably fine given the tags and having to mass update anyway, just wanted to verify -- originally wanted to be sure we could use local settings when looking here)

Reply:
The code is still the same:
https://github.com/GTNewHorizons/GTNHGradle/blob/master/src/main/java/com/gtnewhorizons/gtnhgradle/GTNHSettingsConventionPlugin.java#L28-L35
One of the automatic update steps is taking the specific tag from settings.gradle, and pasting it into gradle.properties to freeze its value - this maintains the old behaviour of keeping a specific version of the spotless configs frozen in time

Reply:
Ah, missed that section.  :+1: